packages/runtime/src/state/makeswift-api-client.ts

```diff
@@ -129,5 +132,20 @@ export type Dispatch = ThunkDispatch<State, unknown, Action>
 export type Store = ReduxStore<State, Action> & { dispatch: Dispatch }
 
 export function configureStore({ serializedState }: { serializedState?: SerializedState }): Store {
-  return createStore(reducer, APIResources.getInitialState(serializedState), applyMiddleware(thunk))
+  const composeEnhancers = composeWithDevToolsDevelopmentOnly({
+    name: `API client store (${new Date().toISOString()})`,
```

Comment:
Because we instantiate the API client in a React hook, multiple copies of this store will be created when running in dev / React Strict Mode; timestamp the store name so we can find the most recent one more easily.